datetime/plugin.spec.yaml

```diff
@@ -266,4 +266,683 @@ actions:
         description: Date in UTC
         type: date
         required: true
-
+  get_future_time:
+    title: Get Future Time
+    description: Calculate a new delta timestamp from parameters and a timestamp
+    input:
+      base_timestamp:
+        title: Base Timestamp
+        description: Timestamp used for calculating a new delta timestamp
```

Comment:
We should  allow both `now` and empty to use the current time and document that here and in the help.md